pkg/manifests/external_dns.go

```diff
@@ -120,11 +168,11 @@ func newExternalDNSClusterRoleBinding(conf *config.Config, dnsConfig *ExternalDn
 	}
 }
 
-func newExternalDNSConfigMap(conf *config.Config, externalDnsConfig *ExternalDnsConfig) (*corev1.ConfigMap, string) {
+func newExternalDNSConfigMap(conf *config.Config, dnsConfig *ExternalDnsConfig) (*corev1.ConfigMap, string) {
 	js, err := json.Marshal(&map[string]interface{}{
-		"tenantId":                    externalDnsConfig.TenantId,
-		"subscriptionId":              externalDnsConfig.Subscription,
-		"resourceGroup":               externalDnsConfig.ResourceGroup,
+		"tenantId":                    dnsConfig.TenantId,
+		"subscriptionId":              dnsConfig.Subscription,
+		"resourceGroup":               dnsConfig.ResourceGroup,
```

Comment:
any reason for these changes? its also externalDnsConfig within other func params

Reply:
thanks for catching that! this went through several iterations and I guess ended up just being the same with different var names.

Changing to standardize the arg name to just be externalDnsConfig